### PR TITLE
Group task

### LINKED ISSUE #11: [1.5pt] Create Item class [group task]
- Create Item class in a separate .rb file.
- All Item class properties visible in the diagram should be defined and set up in the constructor method. Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created.

### LINKED ISSUE #12: [2pt] Implement methods for item class [group task]
1. Implement methods:
- can_be_archived?() in the Item class
- should return true if published_date is older than 10 years
- otherwise, it should return false

2. move_to_archive() in the Item class
- should reuse can_be_archived?() method
- should change the archived property to true if the result of the can_be_archived?() method is true
- should do nothing if the result of the can_be_archived?() method is false

### LINKED ISSUE #13: [2pt] Create a main.rb [group task]
1. Create a main.rb file that will serve as your console app entry-point.
2. Implement startup actions:
   - Present the user with a list of options to perform.
   - Let users choose an option.
   - If needed, ask for parameters for the option.
   - Have a way to quit the app.

### PR DESCRIPTION
**Pull request by rbhatt1999**
# hello Code Reviewer 👋 
image
## Following change done in this pull request:
1. closes #12 
2. closes  #13 
3. closes #11

### CODE REVIEW

**Review by harshitajain06 — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi Team! @rbhatt1999  , 
I'm Harshita.

You did a great job working on this milestone, your task is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of the GitHub projects feature! 💯 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classes/item.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+require 'date'
+class Item
+  attr_accessor :publish_date, :genre, :author, :source, :label
+
+  def initialize(id, publish_date)
+    @id = id
```
Comment (MohamedHNoor) [inline classes/item.rb]: Could you please turn the Id into a Random and let the id in the parameter have a nil value  example `(id: nil) ``@id = id || Random.rand(1..1000)`

**Review by MohamedHNoor — CHANGES_REQUESTED** [review]
### Hi @rbhatt1999  and @harshitajain06, thanks for submitting this pull request!

## Changes Required ♻️ 
I've identified a small issue in this PR  that needs to be addressed before merging:

please check the inline comments I made on the item.rb regarding the id

Happy coding

File: classes/item.rb (hunk 1 above)
Comment (rbhatt1999) [reply classes/item.rb]: issue resolved

**Review by MohamedHNoor — APPROVED** [review]
# STATUS: APPROVED 🟢 👏
## Hi @rbhatt1999 
Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉

### HIGHLIGHTS:
✅  Your repo is professional
✅  Descriptive P.R
✅  No Linters error

Now you can merge this branch and move on 🥇




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!🥇 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**